app/views/content_publisher_feedback_requests/_request_details.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="alert alert-info">
+  <p>Content Publisher is in private beta and supports most of the features that are available on Whitehall.</p>
```

Comment:
Note that I see here we describe "Content Publisher" with title case, whereas the other bits of content in this PR seems to have it as sentence case: "Content publisher" - do we know which way we're supposed to use?

Reply:
All references in the repo now use "Content Publisher".  Checked this with content developers.